generatedocs/data/actions/select/description.md

```diff
@@ -2,18 +2,5 @@
 
 Select random values in an object.
 
-The action supports:
-
-* Listbox
-* Bar chart
-* Scatter plot
-* Map (only the first layer)
-* Combo chart
-* Table
-* Line chart
-* Pie chart
-* Tree map
-* Box plot
-* Distribution plot
-* Histogram
-* Auto chart (including any support generated visualization from this list)
+See [README.md](README.md#limitations) limitations section for limitations to this action.
```

Comment:
Change to "See the [Limitations](README.md#limitations) section in the README.md file for limitations related to this action."